packages/cli/src/deploy/ensureDeployer.ts

```diff
@@ -3,7 +3,7 @@ import { getBalance, getBytecode, sendRawTransaction, sendTransaction, waitForTr
 import deployment from "./create2/deployment.json";
 import { debug } from "./debug";
 
-const deployer = `0x${deployment.address}` as const;
+export const deployer = `0x${deployment.address}` as const;
```

Comment:
this was intentionally not exported because the deployer is configurable by the person deploying

we should make sure to handle that!

Reply:
good catch!!